shared/repository/postgresRepository/user.go

```diff
@@ -36,3 +36,7 @@ func (p *PostgresRepository) UpdateUserPasswordByEmail(email string, password st
 	}
 	return p.db.Model(&models.User{}).Where("email = ?", email).Update("password", user.Password).Error
 }
+
+func (p *PostgresRepository) VerifyUserAccountByEmail(email string) error {
+	return p.db.Model(&models.User{}).Where("email = ?", email).Update("is_verified", true).Error
```

Comment:
For future improvement, since this request is going to be made after the user creation, we can create the token with `user.id` and the query in `db` will be faster

Reply:
DONE